Description: accept newer branch of crate base64
Author: Jonas Smedegaard <dr@jones.dk>
Bug-Debian: https://bugs.debian.org/1084849
Last-Update: 2024-10-09
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -27,7 +27,7 @@
 anyhow = { version = "1.0", features = ["backtrace"] }
 ar = "0.9"
 async-trait = "0.1"
-base64 = "0.21"
+base64 = ">= 0.21, <= 0.22"
 bincode = "1"
 blake3 = "1"
 byteorder = "1.5"
